packages/patternfly-4/react-core/src/components/Modal/ModalBoxCloseButton.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { Button } from '../Button';
+import { TimesIcon } from '@patternfly/react-icons';
+
+export interface ModalBoxCloseButtonProps {
+  /** additional classes added to the close button */
+  className?: string;
+  /** A callback for when the close button is clicked */
+  onClose?(): void;
+}
+
+export const ModalBoxCloseButton: React.FunctionComponent<ModalBoxCloseButtonProps> = ({
+  className = '',
+  onClose = () => undefined,
```

Comment:
I'm surprised the linter allowed this. Change to `onClose = () => undefined as any,`

Reply:
It doesn't allow it after I changed the def of onClose in the props.